app/src/pages/stakingOptions/stakingOptions.tsx

```diff
@@ -223,7 +232,9 @@ export const StakingOptions = (props: { network: string }) => {
       render: (authority) => {
         // TODO Make copyable
         return (
-          <a href={`${explorerStub}account/${authority.toBase58()}`}>{`${authority.toBase58().substring(0, 4)}...`}</a>
+          <a href={Config.explorerUrl(authority.toBase58())} target="_blank" rel="noreferrer">
```

Comment:
Open explorer links in a new tab

Reply:
This was already handled on authorities, added to the name link.